.github/workflows/pytest-dbutils-mysqlfuncts.yml

```diff
@@ -0,0 +1,58 @@
+---
+# This workflow is meant as a foundational workflow for running integration/unit tests on the
+# platform.
+# This workflow also shows the caching mechanisms available for storage
+# and retrieval of cache for quicker setup of test environments.
+
+
+name: Testing mysqlfucnts
+on: [pull_request]
```

Comment:
https://github.community/t/how-to-trigger-an-action-on-push-or-pull-request-but-not-both/16662

Lets use the method here.  So we can do a pull_request to develop/main/releases** and push to develop releases/**

Reply:
test that out on yours and see if that makes gh actions do "one or the other" ish

Reply:
Just tried it out and it appears that GH Actions does do one or the other. 

```md
on:
  push:
    branches:
    - develop
    - releases/**
  pull_request:
    branches:
    - main
    - develop
    - releases/**
```

For example, I created a "releases/foo" branch and created a PR on my personal account. I then pushed a commit and it appears that only one mysqlfuncts Workflow was ran instead of two.

See screenshot: 

![Screen Shot 2021-09-01 at 2 35 05 PM](https://user-images.githubusercontent.com/6901848/131748484-26fbbd00-7673-4235-a471-1968eee14436.png)